backend/internal/server/apis/schedule/games/auto/games.go

```diff
@@ -80,8 +82,9 @@ func handleSave(c *fiber.Ctx) error {
 	return responder.Ok(c)
 }
 
-func handleGenerate(c *fiber.Ctx) error {
-	numberOfGamesPerTeam = 10
+func handleCreateGames(c *fiber.Ctx) error {
+	// TODO need to add number of games per team to the request body and then pass it to the round robin algorithm
+	numberOfGamesPerTeam := 10
```

Comment:
We could also add this to backend/internal/utils/constants

Reply:
true

Reply:
But this really should come from the request